src/importexport/guitarpro/internal/gtp/gp67dombuilder.cpp

```diff
@@ -53,7 +53,6 @@ void GP67DomBuilder::buildGPDomModel(QDomElement* qdomElem)
         assignMap(current);
     }
 
-    std::unordered_map<int, std::shared_ptr<GPRhytm> > rhytms;
```

Comment:
Oh, I should call this out. This random typo is was an unused variable. Fixing the typo of course broke the build since it tripped over the other `rhythms` object which has a totally different shape.